src/components/ResourceItem/tests/ResourceItem.test.tsx

```diff
@@ -465,69 +514,94 @@ describe('<ResourceItem />', () => {
     });
 
     it('renders a checked Checkbox if the item is in the selectedItems context', () => {
-      const wrapper = mountWithAppProvider(
+      const wrapper = mountWithApp(
```

Comment:
This test is in the "SelectMode" describe block, but uses `mockSelectableContext`. Is this intentional?